schemas/core/components/configurator.json

```diff
@@ -89,13 +90,47 @@
     "customerSelection": {
       "type": "object",
       "patternProperties": {
-        "^[a-z0-9\\-]+$": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          }
+        "^.*$": {
```

Comment:
This will match also special characters. Best to stick to only lowercase characters and numbers